src/interaction/sessionrecording.cpp

```diff
@@ -1534,30 +1534,41 @@ void SessionRecording::populateListofLoadedSceneGraphNodes() {
     }
 }
 
-bool SessionRecording::checkIfScriptUsesScenegraphNode(std::string s) {
+void SessionRecording::checkIfScriptUsesScenegraphNode(std::string s) {
     if (s.rfind(scriptReturnPrefix, 0) == 0) {
         s.erase(0, scriptReturnPrefix.length());
     }
     // This works for both setPropertyValue and setPropertyValueSingle
-    if (s.rfind("openspace.setPropertyValue", 0) == 0) {
+    bool containsSetPropertyVal = (s.rfind("openspace.setPropertyValue", 0) == 0);
+    bool containsParensStart = (s.find("(\"") != std::string::npos);
```

Comment:
This could lead to a false negative:
```lua
openspace.setPropertyValue("
openspace.setPropertyValue('
openspace.setPropertyValue([[
```
are all legal and should be checked for

Reply:
Changed the code to handle all of these